### DISCUSSION TITLE
How to load subtitle in Automatic Video Conversion

### DISCUSSION THREAD

**Discussion by MuhammadBayiz** [jlesage/docker-handbrake#337] [Q&A]
As the title says i would like to burn in a subtitle to the video with automatic conversion and i read through the documents all i had was using the hooks but i have one problem, and it's that there's no option for importing a subtitle or simply there's no variable in the hook examples for my use case, could someone clearify what variables are available in the hooks and andy reference to how to use them.

**Comment by jlesage** [comment]
If I understand currently, you would like to load external subtitles ? Usually subtitles used by HandBrake are embedded into the MKV file.

**Reply by MuhammadBayiz** [reply]
Exactly i have external subtitles i want to use them

**Reply by MuhammadBayiz** [reply]
.ass subtitles to be exact. And i want to creat a automatic preset to load those external subtitles(maybe by looking in the same folder as the video source folder for those external subtitles) and burn it to the video
